README.md

```diff
@@ -328,6 +328,40 @@ module "runner" {
 }
 ```
 
+### Scenario: Basic usage on GitLab **>= 16.0.0**
+
+Below is a basic examples of usages of the module if your GitLab instance version is >= 16.0.0.
+
+```hcl
+module "runner" {
+  # https://registry.terraform.io/modules/cattle-ops/gitlab-runner/aws/
+  source  = "cattle-ops/gitlab-runner/aws"
+
+  aws_region  = "eu-west-1"
+  environment = "spot-runners"
+
+  vpc_id                   = module.vpc.vpc_id
+  subnet_ids_gitlab_runner = module.vpc.private_subnets
+  subnet_id_runners        = element(module.vpc.private_subnets, 0)
+
+  runners_name       = "docker-default"
+  runners_gitlab_url = "https://gitlab.com"
+
+  runner_gitlab_access_token_secure_parameter_store_name = "gitlab_token_ssm"
+
+
+  runner_gitlab_registration_config = {
+    type               = "instance_type"
```

Comment:
issue: `instance_type` is not a valid value, is it? `project_type`?

Either `group_id` or `project_id` is missing too.

Reply:
`instance_type` is a valid value and doesn't require neither `group_id` or `project_id` since it is at the instance level.

Reply:
Oh yes, you are right. It's clearly stated in the docs. https://docs.gitlab.com/ee/api/runners.html